### ISSUE TITLE
Unable to access kiosk when my connection is via personal hotspot to my phone while trying to show people stuff at lunch.

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#139]

**Comment by urapadmin** [comment]
please elaborate. You tried to access the online version of kiosk? What happened? etc...

**Comment by lbestock** [comment]
At a restaurant with people I wanted to show our seal impressions to. I got on the free wifi. Kiosk did not open (status bar unmoving after a centimeter of green), though I was able to access other websites (tried with the NYTimes). So I gave myself a hotspot and tried that way, and had the same problem. What I did not try and certainly should have was opening another browser window and going to the online kiosk fresh. Since I tend to have a window open on it I tend to use that. I very often encounter the CSRF protection failure this way. Might my issue have been related? (But it really wouldn't load at all.) I had no problems once I got back to the hotel but cannot remember if I might then have opened a new window. This all from my ipad if that makes any difference.

**Comment by urapadmin** [comment]
it doesn't. I don't know about the free Wifi, but kiosk should load when you use your personal hotspot. Try again. Does it happen here, too? If so, can you open kiosk with your phone? I suspect you were accessing the online kiosk and did not make the mistake of trying to reload a local kiosk without the virtual machine running.

**Comment by lbestock** [comment]
I can access it using my phone as a hotspot while my phone is connected to drahtlos at home. I tested it by having a window open to the file repository in which I performed a search on my iPad while connected to drahtlos. Then I dropped drahtlos on my iPad, picked up the hotspot, and performed another search in the same window without refreshing or anything. It worked just fine. So then I dropped drahtlos on my phone and used my cellular connection, kept the hotspot, and while things were slow, had no other problem. So either it was something with the free wifi at the restaurant or I had been in the local file repository not the online one in that window (but that strikes me as very improbable - I had been in the file repository in the hotel on the iPad away from my own computer most recently and so on the hotel wifi). I will have to try it all again in places with free crappy wifi.
